### ISSUE TITLE
sidebar cannot show forever

### ISSUE THREAD

**Issue by JohnChain** [geany/geany#1753]
Hello everybody

I find the issue that after change the width of sidebar to maybe zero (hold the dragglable right edge of sidebar and draggle it to the very right side of Geany APP window), then I cannot draggle out the sidebar forever. 
Even in full screen mode, after the mouse icon becoming dragglable, I just cannot draggle out the sidebar... 

![screen shot 2018-01-29 at 9 28 06 am](https://user-images.githubusercontent.com/1732598/35489979-17ecf4b4-04d7-11e8-80ec-b6660691b864.png)

Hoping someone can give some help
Johnchain

**Comment by elextr** [comment]
I think this is a theme problem, it doesn't occur here, my theme does not let the handle of the pane disappear so the problem does not occur here.

**Comment by JohnChain** [comment]
@elextr 
Hi elextr
What kind of theme ? 
I never change anything related with theme after installation, purely official release...
Can you share the theme you are using ?

**Comment by elextr** [comment]
The theme depends on the desktop and the system you are using (which you did not specify), here its standard Linux Mint cinnamon dark theme.

**Comment by codebrainz** [comment]
Works for me too.

@JohnChain as a work around you can open `~/.config/geany/geany.conf` in an editor that is **not Geany** and change the value of `treeview_pos` to something more than zero. Make sure you don't use Geany to edit the file. The default Macos text editor should be fine.

@elextr Geany on Macos comes with its own theme embedded in the bundle. There's really no option to change it (aside from totally manually).

**Comment by codebrainz** [comment]
I can reproduce this on Macos. It's because the window manager has an invisible grab area to resize windows and it overlaps where the splitter bar is, sending the drag to the window manager instead of the client.

**Comment by codebrainz** [comment]
@techee maybe for macos, as a workaround, we should limit the GtkPaned positions to N pixels minimum from the edges?

**Comment by JohnChain** [comment]
Hi @codebrainz, it works , thank you 👍 
BTW, this issue will still be there even reinstall Geany APP...

@elextr  thanks too   :)

**Comment by techee** [comment]
Yeah, this happened to me in the past as well. And yes, the fix would be to have some minimum size of the sidebar.

>BTW, this issue will still be there even reinstall Geany APP...

It's because it's stored in the settings file which is preserved when you re-install the app. You have to remove the ~/.config/geany/geany.conf file to get the default settings or edit it as @codebrainz suggested.

**Comment by miwebguy** [comment]
While preference change has worked in the past for me, it doesn't appear to be working.
OpenSUSE Leap 15.4 Gnome 41.2 Geany 1.38
